pyproject.toml

```diff
@@ -31,30 +31,58 @@ skip_empty = true
 #fail_under = 80
 
 [tool.pyright]
-strict = ["starcraft"]
+strict = ["charmcraft"]
 pythonVersion = "3.8"
 pythonPlatform = "Linux"
 venvPath = ".tox"
 venv = "py38"
 
 [tool.mypy]
 python_version = "3.8"
+packages = [
+    "charmcraft",
+]
 exclude = [
     "build",
-    "tests",
     "results",
+    "tests",
+    "tools",
 ]
 warn_unused_configs = true
 warn_redundant_casts = true
 strict_equality = true
-strict_concatenate = true
-warn_return_any = true
+#warn_return_any = true
```

Comment:
should these stay commented?

Reply:
For now yes, as these cause CI failures that'll be fixed with craft-application update. I've put together an issue and premade a PR for it though so the change is trivial once it's ready. https://github.com/canonical/charmcraft/issues/1314